src/app/auth/authentication.service.ts

```diff
@@ -23,6 +23,7 @@ export class AuthenticationService {
     if ('token' in params) {
       this.authToken = params['token'];
       localStorage.setItem('auth_token', this.authToken);
+      location.href = location.protocol + '//' + location.host;
```

Comment:
Did this to get rid of the token from the URL. Need to figure out a better way. Need help!!